src/webhooks/models/webhooksFormatter.js

```diff
@@ -26,12 +27,12 @@ function unknownWebhookEventTypeError(badWebhookEventTypeValue) {
     return new Error(`Unrecognized webhook event: ${badWebhookEventTypeValue}, must be one of the following: ${WEBHOOK_EVENT_TYPES.join(', ')}`);
 }
 
-function getThresholdMessage(state, { isSlack, testName, benchmarkThreshold, lastScores, aggregatedReport, score }) {
+function getThresholdMessage(state, { isSlackOrDiscord, testName, benchmarkThreshold, lastScores, aggregatedReport, score }) {
     let resultText = 'above';
-    let icon = isSlack ? slackEmojis.ROCKET : teamsEmojis.ROCKET;
+    let icon = isSlackOrDiscord ? slackEmojis.ROCKET : teamsEmojis.ROCKET;
     if (state === WEBHOOK_EVENT_TYPE_BENCHMARK_FAILED) {
         resultText = 'below';
-        icon = isSlack ? slackEmojis.CRY : teamsEmojis.CRYING;
+        icon = isSlackOrDiscord ? slackEmojis.CRY : teamsEmojis.CRYING;
     }
     return `${icon} *Test ${testName} got a score of ${score.toFixed(1)}` +
         ` this is ${resultText} the threshold of ${benchmarkThreshold}. ${lastScores.length > 0 ? `last 3 scores are: ${lastScores.join()}` : 'no last score to show'}` +
```

Comment:
instead of using `isSlackOrDiscord`, let's define a dictionary for `slack`, `discord`, and `teams` containing the proper values for emojis. 

It will be cleaner.

Reply:
I create a function where choose the emoji acording with the format
something like this

` module.exports = function (format){
    switch (format) {
        case EVENT_FORMAT_TYPE_SLACK: {
            return slackEmojis;
        }
        case EVENT_FORMAT_TYPE_TEAMS: {
            return teamsEmojis;
        }
        case EVENT_FORMAT_TYPE_DISCORD:{
            return slackEmojis;
        }
        default: {
            throw new Error(`Unrecognized webhook format: ${format}, available options for emojis: Slack, Teams, Discord.`);
        }
    }
};`

what do you think?

Reply:
what do you think about to create a function where choose the emoji acording with the format
something like this

![Screenshot from 2020-10-15 15-27-37](https://user-images.githubusercontent.com/28812225/96192507-05187080-0efb-11eb-9dea-b3e71579e456.png)

and then just define a emoji 

![Screenshot from 2020-10-15 15-29-32](https://user-images.githubusercontent.com/28812225/96192635-46a91b80-0efb-11eb-99df-c4edddf6cd96.png)

Reply:
I am totally fine with that, in the `default` case please use the `EVENT_FORMAT_TYPES` const variable such that we would not have the format types hardcoded in the message